mls-rs-core/Cargo.toml

```diff
@@ -22,6 +22,8 @@ test_suite = ["serde", "dep:serde_json", "dep:itertools"]
 serde = ["dep:serde", "zeroize/serde", "hex/serde", "dep:serde_bytes"]
 last_resort_key_package_ext = []
 post-quantum = []
+self_remove_proposal = []
+rcs_self_remove_proposal = []
```

Comment:
I think it makes sense that we just have an rcs feature that can incorporate anything else that is needed moving forward as well?

Reply:
That makes sense! Based on @mulmarta 's comment above, if we use `0xF003` for the self-remove proposal type in all cases, then we don't need this flag in my PR anymore. Would you prefer that I still add it, and rename it to something more general like `rcs_spec_features` ?

Reply:
Ok, if you don't need it anymore remove it and just note that if we do need this later on it should be general to reduce how many of these flags there are. 

Thanks!

Reply:
Makes sense!